test_util/auth_helper.py

```diff
@@ -0,0 +1,26 @@
+"""
+This module provides a predefined auth token which can be utilized for authentication with the test cluster.
+
+Token valid until 2036 for user albert@bekstil.net
+{
+  "email": "albert@bekstil.net",
+  "email_verified": true,
+  "iss": "https://dcos.auth0.com/",
+  "sub": "google-oauth2|109964499011108905050",
+  "aud": "3yF5TOSzdlI45Q1xspxzeoGBe9fNxm9m",
+  "exp": 2090884974,
+  "iat": 1460164974
+}
+"""
+import os
+
+from pkgpanda import load_json
+
+
+def get_cluster_auth_token():
```

Comment:
This should be called get_ci_oauth_token as to not confuse it with generically getting dcos auth token